app/models/movie.rb

```diff
@@ -0,0 +1,24 @@
+require 'pry'
+class Movie < ApplicationRecord
+  has_many :rentals
+  has_many :customers, through: :rentals
+
+  validates :title, presence: true
+  validates :overview, presence: true
+  validates :release_date, presence: true
+  validates :inventory, presence: true
+
+  # calculates and returns a movie's available inventory
+  # if rental is out, then due_date is a Date object
+  # if rental is back, due_date is nil
```

Comment:
I'm not sure that I agree with this design decision. Having information about the due dates of old rentals might be useful, especially in a real-world version of this app. For the context of this project it works fine, but destroying this information makes me nervous.